tests/server/handlers/test_query_params.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from http import HTTPStatus
+
+from tornado.httpclient import HTTPError
+from tests import utils
+from tornado import testing
```

Comment:
`pytest` is not a standard python library.

```python
from http import HTTPStatus

import pytest
from tornado import testing
from tornado.httpclient import HTTPError

from tests import utils

```

Reply:
While you're fixing up the error message, go ahead and do this, too.